scripts/ubuntu/Readme.md

```diff
@@ -0,0 +1,66 @@
+# This is an easy scripts to run samples from scratch for Ubuntu users.
```

Comment:
Please change the file name to README.md.

Reply:
I'm sorry. Done.